projects/kit/components/slider/slider-options.ts

```diff
@@ -0,0 +1,17 @@
+import {InjectionToken} from '@angular/core';
+import {TuiSizeS} from '@taiga-ui/core';
+
+export interface TuiSliderOptions {
+    readonly size: TuiSizeS;
+    readonly trackColor: string;
+}
+
+export const TUI_SLIDER_DEFAULT_OPTIONS: TuiSliderOptions = {
+    size: 'm',
+    trackColor: 'var(--tui-base-03)',
```

Comment:
I need it to make
```ts
{
    provide: TUI_SLIDER_OPTIONS,
    useValue: {
        ...TUI_SLIDER_DEFAULT_OPTIONS,
        trackColor: 'transparent',
    },
},
```
inside `InputSlider`

Reply:
I think`tuiSliderOptionsProvider` is more better

https://github.com/Tinkoff/taiga-ui/blob/main/projects/addon-commerce/components/money/money-options.ts#L31